src/web/modules/users/users.http.ts

```diff
@@ -8,14 +8,26 @@ import UpdatePhoneNumberFormRequest from './UpdatePhoneNumberForm'
 
 import {formatErrorsForTemplate, ClientFormError} from '../common/validationErrorFormat'
 import {EntityNotFoundError, IOValidationError} from '../../../lib/errors'
+import {User} from "../../../lib/pay-request/services/admin_users/types";
+
+const DEGATEWAY_FLAG = 'degatewayaccountification'
 
 const show = async function show(req: Request, res: Response): Promise<void> {
   const payUser = await AdminUsers.users.retrieve(req.params.id)
-  const context = {payUser, messages: req.flash('info'), csrf: req.csrfToken()}
+  const context = {
+    payUser,
+    accountSimplificationEnabled: isAccountSimplificationEnabled(payUser),
+    messages: req.flash('info'),
+    csrf: req.csrfToken()
+  }
 
   res.render('users/users.show.njk', context)
 }
 
+function isAccountSimplificationEnabled(user: User): boolean {
+  return user.features.includes(DEGATEWAY_FLAG);
+}
```

Comment:
I guess this could be an instance method on the `User` class but it doesn't make much difference

Reply:
Think I'll leave this as is as the `DEGATEWAY_FLAG` is used multiple times in user.http.ts.